contributed_definitions/nyaml/NXcs_cpu_obj.yaml

```diff
@@ -5,7 +5,7 @@ symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
 type: group
-NXcs_cpu_obj(NXobject):
+NXcs_cpu_obj(NXcircuit):
   name(NX_CHAR):
     doc: |
```

Comment:
I do not know what is the useges of NXcircuit here. The says it is the descripton of the CPU, which could be socket, threads per core and architecture of the cpu.

Even, I would vote to delete the cs_cpu base class, if it is not used anywhere. It does not have any spcification for the CPU. This kind of short description `name` and `fabrication` can be write inside the appdef.

Reply:
Agree, followed your suggestion and refactored this set of base classes, only NXcs_computer remains using NXcircuit.